Makefile

```diff
@@ -3,9 +3,9 @@ BUILD_TARGET=./cmd/dbmigrate/*.go
 
 test: build
 	go build -o /dev/null ./examples # verify examples can compile
-	for DATABASE_DRIVER in $(DATABASE_DRIVERS); do (\
-		DATABASE_DRIVER=$$DATABASE_DRIVER bash tests/withdb.sh tests/scenario.sh || exit 1; \
-	); done
+	for DATABASE_DRIVER in $(DATABASE_DRIVERS); do \
+		DATABASE_DRIVER=$$DATABASE_DRIVER bash -euxo pipefail tests/withdb.sh tests/scenario.sh || exit 1; \
+	done
 
```

Comment:
removing extraneous `(` parentheses `)` so that `exit 1` can exit the `make` process